### PR TITLE
docs: update slack config example

### PR DESCRIPTION
**Pull request by inertial-frame**
Tweaked the config.go example:

1. The plugin does not require the `recipients` field to launch. But it does need it when you request a role.
2. I also updated the field to better depict what the values need to be per https://github.com/gravitational/teleport/pull/7041.

This is for issue: https://github.com/gravitational/teleport-plugins/issues/251

### CODE REVIEW

**Review by benarent — APPROVED** [review]

**Review by klizhentas — APPROVED** [review]

File: access/slack/config.go [hunk 1]
```diff
@@ -22,14 +22,21 @@ type SlackConfig struct {
 }
 
 const exampleConfig = `# example slack plugin configuration TOML file
```
Comment (inertial-frame) [inline access/slack/config.go:24]: This is the exact string that's used to generate the example configuration file when running: `teleport-slack configure > teleport-slack.toml` or `teleport-slack configure > /path/to/teleport-slack.toml`. Tested this. 

Decided to include the commands and previous configuration.

**Comment by inertial-frame** [comment]
I made a few more changes here and wanted to get another review pass. Thanks and sorry for the extra review!

**Review by klizhentas — APPROVED** [review]

File: access/slack/INSTALL.md [hunk 2]
```diff
@@ -151,15 +151,22 @@ certificates there and make sure the config points to them.
...
+
+# tctl auth sign --format=tls --auth-server=auth.example.com:3025 --user=access-plugin --out=auth --ttl=1h
+# client_key = "/var/lib/teleport/plugins/slack/auth.key" # Teleport GRPC client secret key ("tls" format")
+# client_crt = "/var/lib/teleport/plugins/slack/auth.crt" # Teleport GRPC client certificate ("tls" format")
+# root_cas = "/var/lib/teleport/plugins/slack/auth.cas"   # Teleport cluster CA certs ("tls" format")
 
 [slack]
-token = "api-token"       # Slack Bot OAuth token
+token = "xoxb-11xx"                                # Slack Bot OAuth token
+recipients = ["person@email.com","YYYYYYY"]        # Optional Slack Rooms
```
Comment (marshall-lee) [inline access/slack/INSTALL.md:169]: `recipients` list is the old mode where recipients are set statically in config file.

Normally, people should specify "suggested reviewers" when they request the role. If any of the suggested reviewers match the slack user emails — that users will be notified.

Comment (marshall-lee) [reply access/slack/INSTALL.md:169]: This is why `recipients` is not a required option.

Comment (inertial-frame) [reply access/slack/INSTALL.md:169]: Ah, I was unaware that's the recommended field to configure. Will update, thanks!

**Pull request merged**